Generate setters for const static member variables

E.g.:

```
/tmp/dartpy/_ZN4dart9optimizer21GradientDescentSolverE.cpp:35:168: error: no viable overloaded '='
.add_static_property("Type", []() { return dart::optimizer::GradientDescentSolver::Type; }, [](const std::string value) { dart::optimizer::GradientDescentSolver::Type = value; })
```

--- a/src/visitor.cpp
+++ b/src/visitor.cpp
@@ -29,7 +29,9 @@ bool IsAssignable(CXXRecordDecl *decl)
 
 bool IsQualTypeAssignable(ASTContext &context, QualType qual_type)
 {
-    if (CXXRecordDecl *decl = qual_type.getTypePtr()->getAsCXXRecordDecl())
+    if (qual_type.isConstQualified())
+        return false;
+    else if (CXXRecordDecl *decl = qual_type.getTypePtr()->getAsCXXRecordDecl())
         return IsAssignable(decl);
     else if (qual_type.getTypePtr()->isArrayType())
         return false;

